### PR TITLE
objects: use provided Transport in deserialization of ENHO+SFPF

### PR DESCRIPTION
**Pull request by filak-sap**
I have been working on RFC enabled functions modules for executing
abapGit operations and I found out that transported objects of type ENHO
and SFPF cannot be pulled over my RFC methods because the specified
transport was not used.

This commit simply takes over the approach from other objects where
we already read the transport number.

For SFPF, I had to add "darkmode" which was missing and which is
necessary to avoid any dialogues.

I am not sure if there are other objects with the same problem, so
I might be posting similar patches in future.

### CODE REVIEW

**Comment by filak-sap** [comment]
OK. Keeping it as a donwstream patch.

**Comment by larshp** [comment]
https://github.com/abapGit/abapGit/pull/5206 contains the solution for a similar downport issue

File: src/objects/zcl_abapgit_object_sfpf.clas.abap [hunk 1]
```diff
@@ -194,15 +194,19 @@ CLASS zcl_abapgit_object_sfpf IMPLEMENTATION.
...
     lo_wb_form ?= load( ).
 
     lv_name = ms_item-obj_name.
+    lv_corrnum = zcl_abapgit_default_transport=>get_instance( )->get( )-ordernum.
 
     TRY.
-        lo_wb_form->delete( lv_name ).
+        lo_wb_form->delete( i_name = lv_name
+                            i_ordernum = lv_corrnum
+                            i_dark = 'X' ).
```
[the remark this answers is not in the document]
Comment (flaiker) [reply src/objects/zcl_abapgit_object_sfpf.clas.abap:209]: @larshp Shouldn't `static_call_via_instance` have caught this?

[the remark this answers is not in the document]
Comment (larshp) [reply src/objects/zcl_abapgit_object_sfpf.clas.abap:209]: yea, its standard, "CL_FP_WB_FORM" is unknown/voided by abaplint

**Pull request closed without merging**